src/cmd/sql_cmd.h

```diff
@@ -1244,12 +1303,26 @@ void HandleSQL(const std::string& sql) {
         }
         case hybridse::node::kPlanTypeFuncDef:
         case hybridse::node::kPlanTypeQuery: {
-            ::hybridse::sdk::Status status;
-            auto rs = sr->ExecuteSQL(db, sql, &status);
-            if (!rs) {
-                std::cout << "ERROR: " << status.msg << std::endl;
+            if (isOnlineMode()) {
+                // Run online query
+                ::hybridse::sdk::Status status;
+                auto rs = sr->ExecuteSQL(db, sql, &status);
+                if (!rs) {
+                    std::cout << "ERROR: " << status.msg << std::endl;
+                } else {
+                    PrintResultSet(std::cout, rs.get());
+                }
             } else {
-                PrintResultSet(std::cout, rs.get());
+                // Run offline query
+                ::openmldb::taskmanager::JobInfo job_info;
+                std::map<std::string, std::string> config;
+                auto status = sr->ExecuteOfflineQuery(sql, config, db, job_info);
+
+                std::vector<::openmldb::taskmanager::JobInfo> job_infos;
+                if (status.OK() && job_info.id() > 0) {
```

Comment:
If status is not ok or job info invalid, should we print the error?

Reply:
We will redefine the output message for not-ok processes laster.